### PR TITLE
doc/rbd/rbd-kubernetes: update for recent ceph-csi versions

### PR DESCRIPTION
**Pull request by idryomov**

### CODE REVIEW

File: doc/rbd/rbd-kubernetes.rst [hunk 1]
```diff
@@ -188,6 +209,7 @@ pool created above, the following YAML file can be used after ensuring that the
         parameters:
            clusterID: b9127830-b0cc-4e34-aa47-9d1a2e9949a8
            pool: kubernetes
+           imageFeatures: layering
```
Comment (Madhu-1) [inline doc/rbd/rbd-kubernetes.rst:212]: can you also add missing controller expand secrets to this storageclass

https://github.com/ceph/ceph-csi/blob/6a46f381c2f5023862d9be80515b41abd273c8bd/examples/rbd/storageclass.yaml#L59-L60

Comment (idryomov) [reply doc/rbd/rbd-kubernetes.rst:212]: It's only needed for volume expansion, right?  And we don't set `allowVolumeExpansion` in this example.

I guess there is no reason to hide that functionality though.  I'll add both.

Comment (Madhu-1) [reply doc/rbd/rbd-kubernetes.rst:212]: Yes, correct. Expansion is supported from long time.

**Review by liewegas — APPROVED** [review]

File: doc/rbd/rbd-kubernetes.rst [hunk 2]
```diff
@@ -188,16 +208,23 @@ pool created above, the following YAML file can be used after ensuring that the
...
            csi.storage.k8s.io/node-stage-secret-name: csi-rbd-secret
            csi.storage.k8s.io/node-stage-secret-namespace: default
         reclaimPolicy: Delete
+        allowVolumeExpansion: true
         mountOptions:
            - discard
         EOF
         $ kubectl apply -f csi-rbd-sc.yaml
 
+Note that in Kubernetes v1.14 and v1.15 volume expansion feature was in alpha
```
Comment (humblec) [inline doc/rbd/rbd-kubernetes.rst:225]: Actually we dont need this mention, considering the mentioned versions of Kube is really old or `< n-3 `

Comment (idryomov) [reply doc/rbd/rbd-kubernetes.rst:225]: Well, the doc covers v1.13+, so I thought it wouldn't hurt.

**Pull request merged**